pkgs/top-level/beam-packages.nix

```diff
@@ -37,6 +37,12 @@ in
       exdoc = self.packages.erlang_26.ex_doc;
       inherit wxSupport systemdSupport;
     };
+    erlang_27_odbc = self.interpreters.erlang_27.override { odbcSupport = true; };
+    erlang_27_javac = self.interpreters.erlang_27.override { javacSupport = true; };
+    erlang_27_odbc_javac = self.interpreters.erlang_27.override {
+      javacSupport = true;
+      odbcSupport = true;
+    };
```

Comment:
```suggestion
```
We are removing this at #317311
If we add it like that hydra will be building it. But it's not necessary. It's not used often.